src/config.js

```diff
@@ -14,17 +14,46 @@ require('dotenv').config();
 const { from } = require('env-var');
 const yaml = require('js-yaml');
 
-function getFileContent(path) {
+function getFileContent (path) {
     if (!fs.existsSync(path)) {
         throw new Error('File doesn\'t exist');
     }
     return fs.readFileSync(path);
 }
 
+/**
+     * Gets Resources versions from enviromental variable RESOURCES_VERSIONS
+     * should be string in format: "resouceOneName=1.0,resourceTwoName=1.1"
+     */
+function getVersionFromConfig (resourceString) {
+    const resourceVersionMap = {};
+    resourceString
+        .split(',')
+        .forEach(e => e.split('=')
+            .reduce((p, c) => {
+                resourceVersionMap[p] = {
+                    contentVersion: c,
+                    acceptVersion: c.split('.')[0],
+                };
+            }));
+    return resourceVersionMap;
+}
+    
+function validateResourceVersions (resourceString) {
```

Comment:
I would rename this to `parseResourceVersions`, since it's both validating and parsing.